README.md

```diff
@@ -1,2 +1,6 @@
 trytond-country-sequence
 ========================
+[![Build Status](https://api.travis-ci.org/openlabs/trytond-country-sequence.svg?branch=develop)](https://travis-ci.org/openlabs/trytond-country-sequence/)
```

Comment:
In markdown syntax, you keep one line blank and then start from the next line.

By extra lines I meant that there were two lines which were blank instead of the usual one.

Ref: [Tutorial](http://daringfireball.net/projects/markdown/basics)